Mathlib/NumberTheory/Harmonic/ZetaAsymp.lean

```diff
@@ -347,4 +352,71 @@ lemma _root_.isBigO_riemannZeta_sub_one_div {F : Type*} [Norm F] [One F] [NormOn
 
 end continuity
 
+section val_at_one
+
+open Complex
+
+lemma tendsto_Gamma_term_aux : Tendsto (fun s ↦ 1 / (s - 1) - 1 / Gammaℝ s / (s - 1)) (𝓝[≠] 1)
+    (𝓝 (-(γ + Complex.log (4 * ↑π)) / 2)) := by
+  have h := hasDerivAt_Gammaℝ_one
+  rw [hasDerivAt_iff_tendsto_slope, slope_fun_def_field, Gammaℝ_one] at h
+  have := h.div (hasDerivAt_Gammaℝ_one.continuousAt.tendsto.mono_left nhdsWithin_le_nhds)
+    (by rw [Gammaℝ_one]; exact one_ne_zero)
```

Comment:
```suggestion
    (Gammaℝ_one.trans_ne one_ne_zero)
```

Reply:
That's a handy lemma, I didn't know about that one :smile: